src/views/view-utils.js

```diff
@@ -1,6 +1,25 @@
 (function () {
   'use strict';
 
+  /**
+   * @ngdoc object
+   * @name patternfly.views.pfViewUtils
+   * @description
+   * A utility constant to return view objects used for Dashboard, Card, Table, List, and Topology view switcher types.
+   * @example
+   * <pre>
+   * $scope.viewsConfig = {
+   *   views: [pfViewUtils.getListView(), pfViewUtils.getCardView(), pfViewUtils.getTableView()]
+   * };
+   * </pre>
+   * Each getXxxxView() returns an object:
+   * <ul style='list-style-type: none'>
+   *    <li>.id - (String) Unique id for the view, used for comparisons
+   *    <li>.title - (String) Optional title, uses as a tooltip for the view selector
+   *    <li>.iconClass - (String) Icon class to use for the view selector
+   * </ul>
+   * Please see {@link patternfly.toolbars.component:pfToolbar pfToolbar} for use in Toolbar View Switcher
```

Comment:
Like this... there's mention of 
> Dashboard, Card, Table, List, and Topology view switcher types

Any aspect of those that could also be linked?

Reply:
List, Card, and Table view switchers are represented in the pfToolbar link.  We don't have examples for Dashboard and Topology.